tests/functional/lib/helpers.js

```diff
@@ -859,6 +859,56 @@ define([
       .then(testElementExists('.attached' + attachedId));
   });
 
+  /**
+   * Store the data sent for a WebChannel event into sessionStorage.
+   *
+   * @param {string} expectedCommand command to store data for.
+   * @returns {promise}
+   */
+  const storeWebChannelMessageData = thenify(function (expectedCommand) {
+    return this.parent
+      .executeAsync(function (expectedCommand, callback) {
+        function listener(e) {
+          var command = e.detail.message.command;
+          if (command === expectedCommand) {
+            const storedEvents = JSON.parse(sessionStorage.getItem('webChannelEventData')) || {};
+            storedEvents[command] = e.detail.message;
```

Comment:
It's a good question - I asked myself the same thing because the `fxaccounts:can_link_account` message can be sent multiple times. I couldn't convince myself what was best, so I took the easy approach.